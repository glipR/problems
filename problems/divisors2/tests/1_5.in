48373830_941