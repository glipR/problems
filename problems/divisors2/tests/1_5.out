1124149238039
